esp/templates/users/userchecklist.html

```diff
@@ -14,6 +14,19 @@
 <br />
 <br />
 
+<script type="text/javascript">
```

Comment:
I tested this out -- the behavior if you do "select all", then uncheck a few, then change your mind and remove "select all" is very confusing.  I'd suggest setting everything to the new state of the "select all" checkbox, rather than toggling all the other checkboxes.

Reply:
Great, much simpler and better behavior.  Thanks!